source/clients/admin/microkernel_architecture/microkernel_modules/functionalities_manager/FunctionalitiesManager.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef FUNCTIONALITIES_MANAGER_HPP_
+# define FUNCTIONALITIES_MANAGER_HPP_
+
+# include "AFunctionalitiesManager.hpp"
+
+class FunctionalitiesManager : public AFunctionalitiesManager
+{
+public:
+    FunctionalitiesManager(QObject *parent = 0);
```

Comment:
It may be a good thing, in a c++ 11 way, to forbid the use of `NULL` and to replace it with `nullptr` in our project
But we need to check the real use/interest of that
We'll discuss that later